docs/getting-started/products/ybribe/how-it-works.md

```diff
@@ -0,0 +1,78 @@
+# How it Works
+ 
+
+## Scenario #1: Voters Looking to Claim Bribes
+
+veCRV holders can sell their gauge votes to the highest bidder each week by 1- voting on the bribed gauge with the highest APR and then choosing to 2- claim any applicable rewards
+
+![](https://i.imgur.com/uwO1vBe.png)
+
+***Voting on a Briber's Guage***
+
+
+1. Review the APRs for each guage.
+
+![](https://i.imgur.com/5Qu240a.png)
+
+*    Current Period APR: The APR users can claim this week (if they haven’t already claimed)
+*    Pending Period APR: The APR they will be able to claim next week (week in this case starts on Thursday)
+    
+
+2. Use the APRs to identify which guage to vote for.
+3. Head to Curve by clicking **"Vote for Guage"**  at the top of the page to cast your vote. 
+
+![](https://i.imgur.com/mDUm09z.png)
+
+
+
+***Claiming Bribes***
+
+4. After you vote, you’ll be able to claim any applicable rewards at the start of the next period.
+
+![](https://i.imgur.com/iHnEpZk.png)
+
+**Claim periods** run from Thursday to Thursday, and the scrolling clock on the main page displays the time remaining in the current period.
+
+> **Note**: Make sure you claim any claimable rewards before the current period ends, and/ or submit your votes for the next period. Be sure to claim any rewards before voting again on the same gauge or you’ll be locked out of rewards until the following week.
+> 
+## Example
+
+Facu is a veCRV holder. Facu goes to yBribe (which gets all data directly on-chain) to figure out which gauge is the best one to vote for.
+
+![](https://i.imgur.com/OR1ZDlK.png)
+
+As seen above, we can see MIM gauge is the best for the upcoming period. Facu goes to vote for the MIM gauge on Curve and then at the start of the next period, they can claim their bribe.
+
+The amount of the overall bribe they will collect is dynamically calculated based on how much vote weight Facu provided relative to global vote weight on MIM gauge.
+
+----
+
+## Scenario #2: Offering a Bribe
+
+Bribers can offer a bribe to increase CRV emissions to your favorite Curve pool. Bribers can deposit any token as their bribe.
+
+![](https://i.imgur.com/tLp1787.png)
+
+
+1. Toggle over to the "Offer a Bribe" section of yBribe.
+2. Weight, Current $/veCRV, and Pending $/veCRV are listed for each token.
+3. Offer bribe for upcoming period
+
+
+![](https://i.imgur.com/5O5UT8j.png)
+
+
+yBribe now lets users assign rewards to a different claiming address than the one used to vote. It also fixes a bug which under certain circumstances, allowed an initial bribe allocation to be drained before the start of a voting period.
```

Comment:
allowed an initial bribe allocation to be drained before the start of a voting period.

Was there a security release for this or no since it wasn't a yearn contract to start with? link to more info?

Reply:
It also fixes a bug `that` under certain circumstances...